src/statmodels.jl

```diff
@@ -214,11 +282,18 @@ Return the fitted values of the model.
 fitted(obj::RegressionModel) = error("fitted is not defined for $(typeof(obj)).")
 
 """
-    model_response(obj::RegressionModel)
+    response(obj::RegressionModel)
 
 Return the model response (a.k.a. the dependent variable).
 """
-model_response(obj::RegressionModel) = error("model_response is not defined for $(typeof(obj)).")
+response(obj::RegressionModel) = error("response is not defined for $(typeof(obj)).")
+
+"""
+    meanresponse(obj::RegressionModel)
+
+Return the mean of the response.
+"""
+meanresponse(obj::RegressionModel) = error("meanresponse is not defined for $(typeof(obj)).")
```

Comment:
Could you explain why this is useful? Also I guess we could have a `mean(response(obj))` fallback?

Reply:
I was thinking of GLM frameworks where the `link` might require the mean response. I also didn't want to use the fallback due to potentially using a wrong value for models with weights.

Reply:
Good point about weights. What do you mean exactly about the link?

Reply:
For example, `GLM.linkfun(L::Link, μ)`.

Reply:
But in what context do you need to call it? Couldn't we provide an API to call the link on the mean response instead?

Reply:
There was a request to include a `link` argument to `response` from the people at GLM. However, I think that requires that the `link` code be ported to a small package or StatsBase. Other instances of when one might want the `meanresponse` include estimation for intercept term, and `mss`.